source/data-formats/bson.txt

```diff
@@ -40,6 +37,27 @@ The code samples in this guide use the following BSON document as an example:
        "name" : "Mongo's Pizza"
    }
 
+BSON Data Types
+---------------
+
+BSON supports all JSON data structure types and adds support for types including 
+dates, different size integers, ``ObjectId``, and binary data. For a complete list of 
+supported types, see the :manual:`BSON Types </reference/bson-types>` page in the 
+{+mdb-server+} Manual.
+
+Universally Unique IDs (UUIDs)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The {+driver-short+} supports UUIDs by using the BSON Binary subclass ``UUID``. You 
+can create a ``UUID`` object by using the ``UUID()`` method. The following code 
+example generates a random UUID:
+
+.. code-block:: javascript
+
+   import { BSON } from 'mongodb';  
+
+   const myUuid = new BSON.UUID();
```

Comment:
```suggestion
   import { UUID } from 'mongodb';  

   const myUuid = new UUID();
```

Not wrong, but most folks would skip past the namespace. There aren't a lot of `new BSON.ObjectId()` examples out there.